### DISCUSSION TITLE
Release notes for 1.7.26

### DISCUSSION THREAD

**Discussion by schmod** [swc-project/swc#9566] [General]
Are there any release notes for the latest release (1.7.26)?  

Generally speaking, it can be a warning-sign to see releases published without any documentation (historically this has been a telltale of a supply-chain attack).  

Is there a way that we can confirm that it's authentic?

**Comment by kdy1** [comment]
Now changelog will be updated by CI while runing `cargo bump` (swc-specific crate publish command) 

See https://github.com/swc-project/swc/commit/c6cc9b65880e5e2aa4fb26157ace45f6e8004c4a

**Reply by kdy1** [reply]
It seems like https://github.com/swc-project/swc/actions/runs/10808039820/job/29983240425 didn't push the tag to GitHub

**Reply by kdy1** [reply]
Well there's a git tag so it's a bug of git cliff
<img width="1840" alt="image" src="https://github.com/user-attachments/assets/3a5c96ae-fb37-4d6a-90d2-6226a15a3132">
